src/MetadataUtility/Metadata/WildlifeAcoustics/WamdExtractor.cs

```diff
@@ -0,0 +1,78 @@
+// <copyright file="WamdExtractor.cs" company="QutEcoacoustics">
+// All code in this file and all associated files are the copyright and property of the QUT Ecoacoustics Research Group.
+// </copyright>
+
+namespace MetadataUtility.Metadata.WildlifeAcoustics
+{
+    using System.Threading.Tasks;
+    using MetadataUtility.Audio;
+    using MetadataUtility.Models;
+    using Microsoft.Extensions.Logging;
+
+    public class WamdExtractor : IMetadataOperation
+    {
+        private readonly ILogger<WamdExtractor> logger;
+
+        public WamdExtractor(ILogger<WamdExtractor> logger)
+        {
+            this.logger = logger;
+        }
+
+        public ValueTask<bool> CanProcessAsync(TargetInformation information)
+        {
+            var result = information.IsPcmWaveFile() && information.HasVersion1WamdChunk();
```

Comment:
Theoretically a .wac file could also have a wamd chunk. Also would be true for WA new proprietary file format (which I forget the name of). Leave a note for future editiors?

// TODO: add support for .wac (or other WA) files?